tests/unit/src/test/scala/tests/DetectionSuite.scala

```diff
@@ -20,9 +21,24 @@ object DetectionSuite extends BaseSuite {
   def checkNotSbt(name: String, layout: String): Unit = {
     checkSbt(name, layout, isTrue = false)
   }
+
   def checkSbt(name: String, layout: String, isTrue: Boolean = true): Unit = {
     test(s"sbt-$name") {
-      check(layout, new BuildTools(_, Nil).isSbt, isTrue)
+      check(
+        layout,
+        p => BuildTools.default(p).isSbt,
+        isTrue
+      )
+    }
+  }
+
+  def checkPants(name: String, layout: String, isTrue: Boolean = true): Unit = {
+    test(s"pants-$name") {
+      check(
+        layout,
+        p => BuildTools.default(p).isPants,
```

Comment:
should this commit be in the Pants PR? (not a big deal anyway)

Reply:
It should be but it doesn't matter much, `BuildTools` already has logic to detect a Bazel workspace in order to display a message that "bazel is not supported"

Reply:
Ah, it makes sense 👍